governance/xc_admin/packages/proposer_server/package.json

```diff
@@ -21,11 +21,14 @@
   "dependencies": {
     "@coral-xyz/anchor": "^0.29.0",
     "@pythnetwork/client": "^2.17.0",
+    "@pythnetwork/xc-admin-common": "*",
     "@solana/web3.js": "^1.76.0",
     "@sqds/mesh": "^1.0.6",
     "@types/cors": "^2.8.17",
     "cors": "^2.8.5",
-    "ts-node": "^10.9.1",
-    "@pythnetwork/xc-admin-common": "*"
+    "ts-node": "^10.9.1"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.21"
```

Comment:
for some reason I was getting this error even though this package wasnt changed and hence had to add this

```
    ✖  @pythnetwork/xc-admin-proposer-server:build
       > @pythnetwork/xc-admin-proposer-server@0.0.0 build
       > tsc

       src/index.ts(1,44): error TS7016: Could not find a declaration file for module 'express'. '/Users/danielchew/Code/pyth-crosschain/node_modules/express/index.js' implicitly has an 'any' type.
         Try `npm i --save-dev @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`
       npm ERR! Lifecycle script `build` failed with error:
       npm ERR! Error: command failed
       npm ERR!   in workspace: @pythnetwork/xc-admin-proposer-server@0.0.0
       npm ERR!   at location: /Users/danielchew/Code/pyth-crosschain/governance/xc_admin/packages/proposer_server
```